web-app/src/app/tabs/battery/battery.component.html

```diff
@@ -1,207 +1,212 @@
 <div class="battery">
 	<div class="header">
-		<span class="aliveGroup">
-	    		<div *ngIf="battery.batteryAlive; else elseBlock"><img class="circleAlive" src="../../../assets/GreenCircle.png" alt="Red Circle" id="circle"></div>
-	    		<ng-template #elseBlock><img class="circleAlive" src="../../../assets/RedCircle.png" alt="Red Circle" id="circle"></ng-template>
-				<span class="aliveText">Alive</span>
+		<span class="alive-group">
```

Comment:
I'm just going to assume this file works but it would be nice if it was a bit more readable hahaha